Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.16.7-alpine3.14 AS gobuild
+FROM golang:1.17.7-alpine3.15 AS gobuild
```

Comment:
The go version had to be updated to `1.17` or higher, because the new `libp2p` modules require so.